src/gen/test_generators.rs

```diff
@@ -223,6 +223,7 @@ fn run_generator(#[case] name: &str) {
             Err(code) => (0, format!("FAILED: {}\n", u32::from(code.1))),
         };
 
+        let mut a = make_allocator(*flags);
```

Comment:
this was an oversight earlier. not creating a new allocator when running the test again meant we just doubled the number of allocations. With this new limit, we exceed the limit the second time we run it, causing a discrepancy. We expect the output to be the same. This fixes it by starting both runs with fresh allocators.